core/src/main/java/bisq/core/api/CorePaymentAccountsService.java

```diff
@@ -145,7 +145,6 @@ synchronized PaymentAccount createCryptoCurrencyPaymentAccount(String accountNam
         Optional<CryptoCurrency> cryptoCurrency = getCryptoCurrency(currencyCode.toUpperCase());
         cryptoCurrency.ifPresent(cryptoCurrencyAccount::setSingleTradeCurrency);
         user.addPaymentAccount(cryptoCurrencyAccount);
-        if (!(cryptoCurrencyAccount instanceof AssetAccount)) accountAgeWitnessService.publishMyAccountAgeWitness(cryptoCurrencyAccount.getPaymentAccountPayload()); // TODO (woodser): applies to Haveno?
```

Comment:
Why is this line removed? (The TODO can be removed since the account age witness has been integrated.)

Reply:
sorry reverted without the TODO

Reply:
and actually.. if (!(cryptoCurrencyAccount instanceof AssetAccount)) is always true.. ehm false